app/reducers/dataFeeds.test.js

```diff
@@ -32,28 +35,82 @@ describe('dataFeeds', () => {
   describe('DATA_FEED_REMOVED_EVENT', () => {
 
     it('removes the data feed with the matching address', () => {
-      const actual = dataFeeds([mockAddr(0)], dataFeedRemovedAction(mockAddr(0)))
+      const actual = dataFeeds([mockDataFeed(0)], dataFeedRemovedAction(mockDataFeed(0)))
       const expected = []
       assert.deepEqual(actual, expected)
     })
 
     it('does not remove any data feeds if there is no matching address', () => {
-      const actual = dataFeeds([mockAddr(1)], dataFeedRemovedAction(mockAddr(0)))
-      const expected = [mockAddr(1)]
+      const actual = dataFeeds([mockDataFeed(1)], dataFeedRemovedAction(mockDataFeed(0)))
+      const expected = [mockDataFeed(1)]
       assert.deepEqual(actual, expected)
     })
 
   })
+
+  describe('DATA_FEED_LATEST_RESULT_LOADED', () => {
+    let currentResult, lastUpdated, address, nullLastUpdated
+
+    beforeEach(async () => {
+      nullLastUpdated = 0
```

Comment:
would recommend using "expected" and "actual" language in the variable naming so it's easier to see what's what. `const expectedResult = uintToBytes32(123)` for example